In media search results, rotating screen triggers crash

Crash in current master.
Steps:
1. Perform a media search
2. Rotate screen
3. Crash:
```
07-27 13:55:23.731 24931 24931 I Choreographer: Skipped 170 frames!  The application may be doing too much work on its main thread.
07-27 13:55:23.817  1916  2413 I Icing   : updateResources: need to parse xzh{com.google.android.gms}                      
07-27 13:55:23.904   937   937 I GoogleInputMethod: onStartInput() : Dummy InputConnection bound 
07-27 13:55:23.904   937   937 I ShortcutsDataManager: startImportContentTask()        
07-27 13:55:23.904   937   937 I ShortcutsDataManager: onImportAborted()      
07-27 13:55:24.214  1916  2413 I Icing   : Performing maintenance usage 29446141 budget 1824176126 free 98.386% index free 92.446% purge? false target 272874388                                                                   
07-27 13:55:24.218 24931 24931 W System.err: io.reactivex.exceptions.OnErrorNotImplementedException: Attempt to invoke virtual method 'void android.view.View.setVisibility(int)' on a null object reference                                            
07-27 13:55:24.247 24931 24931 W System.err:    at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:704)
07-27 13:55:24.247 24931 24931 W System.err:    at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:701)                                        
07-27 13:55:24.247 24931 24931 W System.err:    at io.reactivex.internal.observers.LambdaObserver.onError(LambdaObserver.java:74)   
07-27 13:55:24.247 24931 24931 W System.err:    at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:64)
07-27 13:55:24.247 24931 24931 W System.err:    at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:200)
07-27 13:55:24.248 24931 24931 W System.err:    at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:252)
07-27 13:55:24.248 24931 24931 W System.err:    at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:109)
07-27 13:55:24.248 24931 24931 W System.err:    at android.os.Handler.handleCallback(Handler.java:751)                                                                                
07-27 13:55:24.248 24931 24931 W System.err:    at android.os.Handler.dispatchMessage(Handler.java:95)                              
07-27 13:55:24.248 24931 24931 W System.err:    at android.os.Looper.loop(Looper.java:154)          
07-27 13:55:24.248 24931 24931 W System.err:    at android.app.ActivityThread.main(ActivityThread.java:6186)       
07-27 13:55:24.248 24931 24931 W System.err:    at java.lang.reflect.Method.invoke(Native Method)
07-27 13:55:24.248 24931 24931 W System.err:    at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)   
07-27 13:55:24.248 24931 24931 W System.err:    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
07-27 13:55:24.252 24931 24931 W System.err: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.setVisibility(int)' on a null object reference
07-27 13:55:24.252 24931 24931 W System.err:    at fr.free.nrw.commons.explore.images.SearchImageFragment.updateImageList(SearchImageFragment.java:124)
07-27 13:55:24.252 24931 24931 W System.err:    at fr.free.nrw.commons.explore.SearchActivity.lambda$setTabs$1$SearchActivity(SearchActivity.java:111)
07-27 13:55:24.252 24931 24931 W System.err:    at fr.free.nrw.commons.explore.SearchActivity$$Lambda$1.accept(Unknown Source)                                                        
07-27 13:55:24.252 24931 24931 W System.err:    at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:60)    
07-27 13:55:24.253 24931 24931 W System.err:    ... 10 more                                         
07-27 13:55:24.271 24931 24931 E AndroidRuntime: FATAL EXCEPTION: main                                             
07-27 13:55:24.271 24931 24931 E AndroidRuntime: Process: fr.free.nrw.commons.debug, PID: 24931                                      
07-27 13:55:24.271 24931 24931 E AndroidRuntime: io.reactivex.exceptions.OnErrorNotImplementedException: Attempt to invoke virtual method 'void android.view.View.setVisibility(int)' on a null object reference
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:704)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at io.reactivex.internal.functions.Functions$OnErrorMissingConsumer.accept(Functions.java:701)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at io.reactivex.internal.observers.LambdaObserver.onError(LambdaObserver.java:74)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:64)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:200)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:252)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:109)                          
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:751)                      
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:95)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:154)                 
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6186)                                                                                        
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)    
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
07-27 13:55:24.271 24931 24931 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.setVisibility(int)' on a null object reference
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at fr.free.nrw.commons.explore.images.SearchImageFragment.updateImageList(SearchImageFragment.java:124)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at fr.free.nrw.commons.explore.SearchActivity.lambda$setTabs$1$SearchActivity(SearchActivity.java:111)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at fr.free.nrw.commons.explore.SearchActivity$$Lambda$1.accept(Unknown Source)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:60)
07-27 13:55:24.271 24931 24931 E AndroidRuntime:        ... 10 more                               
07-27 13:55:24.303   784  1111 W ActivityManager:   Force finishing activity fr.free.nrw.commons.debug/fr.free.nrw.commons.explore.SearchActivity
```

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -114,6 +114,7 @@
         <activity
             android:name=".explore.SearchActivity"
             android:label="@string/title_activity_search"
+            android:configChanges="orientation|keyboardHidden"
             android:parentActivityName=".contributions.ContributionsActivity"
             />
 

--- a/app/src/main/java/fr/free/nrw/commons/explore/SearchActivity.java
+++ b/app/src/main/java/fr/free/nrw/commons/explore/SearchActivity.java
@@ -1,5 +1,6 @@
 package fr.free.nrw.commons.explore;
 
+import android.content.res.Configuration;
 import android.database.DataSetObserver;
 import android.os.Bundle;
 import android.support.design.widget.TabLayout;
@@ -193,6 +194,10 @@ public class SearchActivity extends NavigationBaseActivity implements MediaDetai
         }
         mediaDetails.showImage(index);
         forceInitBackButton();
+    }
+    @Override
+    public void onConfigurationChanged(Configuration newConfig) {
+        super.onConfigurationChanged(newConfig);
     }
 
     /**

